orderbook/src/main.rs

```diff
@@ -87,6 +87,15 @@ struct Arguments {
     /// thinks they are bad. Base tokens are automatically allowed.
     #[structopt(long, env = "ALLOWED_TOKENS", use_delimiter = true)]
     pub allowed_tokens: Vec<H160>,
+
+    /// The amount of time a classification of a token into good or bad is valid for.
+    #[structopt(
+        long,
+        env = "BLOCK_STREAM_POLL_INTERVAL_SECONDS",
```

Comment:
So we were discussion/considering removing the `env = "LONG_CAPS"` stuff, but here would be an example scenario where it is necessary because its actually different than the var name. Will keep in mind when submitting the PR for this.